src/google/iam.js

```diff
@@ -25,7 +42,7 @@ async function getServiceAccount(project, name, auth) {
       timeout: 2000,
```

Comment:
related to previous comment: 

`timeout` is a deprecated `helix-fetch` extension. The standard way is to use `AbortController`:

```javascript
  const { fetch, AbortController, AbortError } = require('@adobe/helix-fetch');

  const controller = new AbortController();
  setTimeout(() => controller.abort(), 1000);
  const { signal } = controller;

  try {
    const resp = await fetch('https://httpbin.org/json', { signal });
    const jsonData = await resp.json();
  } catch (err) {
    if (err instanceof AbortError) {
      console.log('fetch timed out after 1s');
    }
  }
```

or the `helix-fetch` convenience helper `timeoutSignal(ms)`:

```javascript
  const { fetch, timeoutSignal, AbortError } = require('@adobe/helix-fetch');

  try {
    const resp = await fetch('https://httpbin.org/json', { signal: timeoutSignal(1000) });
    const jsonData = await resp.json();
  } catch (err) {
    if (err instanceof AbortError) {
      console.log('fetch timed out after 1s');
    }
  }
```